edd-variable-pricing-descriptions.php

```diff
@@ -67,6 +67,24 @@ function edd_vpd_download_price_table_row( $post_id, $key, $args ) {
 <?php }
 add_action( 'edd_download_price_table_row', 'edd_vpd_download_price_table_row', 10, 3 );
 
+/**
+ * Adds the description input field for the EDD >= 3.0.
+ *
+ * @since 1.0.4
+ */
+function edd_vpd_download_price_option_row( $post_id, $key, $args ) {
+	$description = isset($args['description']) ? $args['description'] : null;
+	?>
+	<div class="edd-custom-price-option-section edd-variable-pricing">
+		<span class="edd-custom-price-option-section-title"><?php echo esc_html( 'Option Description', 'edd-vpd' ); ?></span>
+		<div class="edd-custom-price-option-section-content edd-form-row">
+			<input type="text" class="edd_variable_prices_description" value="<?php echo esc_attr( $description ); ?>" placeholder="<?php echo esc_attr( 'Option Description', 'edd-vpd' ); ?>" name="edd_variable_prices[<?php echo esc_attr( $key ); ?>][description]" id="edd_variable_prices[<?php echo esc_attr( $key ); ?>][description]" size="20" style="width:100%" />
```

Comment:
```suggestion
			<input type="text" class="edd_variable_prices_description large-text" value="<?php echo esc_attr( $description ); ?>" placeholder="<?php esc_html_e( 'Option Description', 'edd-vpd' ); ?>" name="edd_variable_prices[<?php echo esc_attr( $key ); ?>][description]" id="edd_variable_prices[<?php echo esc_attr( $key ); ?>][description]" />
```
Several changes there but note that using a Core WP input class can usually save having to do inline size/styles.